apps/dapp/pages/home.tsx

```diff
@@ -7,27 +7,43 @@ import { serverSideTranslations } from '@dao-dao/i18n/serverSideTranslations'
 
 import {
   FeaturedDAOsList,
+  FeaturedDao,
   PinnedDAOsList,
   PinnedProposalsList,
   SmallScreenNav,
 } from '@/components'
 
-const HomePage: NextPage = () => (
+interface HomePageProps {
+  featuredDaos: FeaturedDao[]
+}
+
+const HomePage: NextPage<HomePageProps> = ({ featuredDaos }) => (
   <>
     <SmallScreenNav />
 
     <div className="p-4 space-y-6 max-w-6xl md:p-6">
       <PinnedProposalsList />
       <PinnedDAOsList />
-      <FeaturedDAOsList />
+      <FeaturedDAOsList featuredDaos={featuredDaos} />
     </div>
   </>
 )
 
 export default HomePage
 
-export const getStaticProps: GetStaticProps = async ({ locale }) => ({
-  props: {
-    ...(await serverSideTranslations(locale, ['translation'])),
-  },
-})
+export const getStaticProps: GetStaticProps = async ({ locale }) => {
+  const resp = await fetch(
+    'https://dao-stats.withoutdoing.com/mainnet/featured_daos.json'
+  )
+  // These are returned as a timeseries in the form [{time, value}, ...].
+  const featuredDaosOverTime = await resp.json()
```

Comment:
hmm.. i think this is fine for now, but we may want to rethink this since i don't think old data is ever shown on the page, so we are dl'ing extra data we never use. but i think it's fine for now 👍

Reply:
Agree. We can update the generation script later if needed. :)